src/avfoundation.cs

```diff
@@ -12996,7 +13485,7 @@ interface AVAssetDownloadStorageManager
 		AVAssetDownloadStorageManagementPolicy GetStorageManagementPolicy (NSUrl downloadStorageUrl);
 	}
 
-	[NoWatch, NoTV, NoMac, iOS (11,0)]
+	[NoWatch, NoTV, Mac (12,0), iOS (11,0)]
```

Comment:
catalyst ?

Reply:
same, it is the min version of catalyst.